docs/_includes/faq.html

```diff
@@ -11,7 +11,7 @@ <h2 class="accordion-header" id="flush-headingOne">
                     </button>
                   </h2>
                   <div id="flush-collapseOne" class="accordion-collapse collapse" aria-labelledby="flush-headingOne" data-parent="#accordionExample">
-                    <div class="accordion-body">Yes, certainly. As a matter of fact, you can leverage GetNighthawk’s build workflows as the basis for your own. We recommend collaborating in the GetNighthawk project to improve upon the workflows to meet your specific needs. Join the community and let us know.</div>
+                    <div class="accordion-body">Yes, certainly. As a matter of fact, you can leverage Nighthawk’s build workflows as the basis for your own. We recommend collaborating in the Nighthawk project to improve upon the workflows to meet your specific needs. Join the community and let us know.</div>
```

Comment:
@toth2000 Thanks for taking this up :)
The FAQ's are not unique questions but instead all the answers are duplicates of each other. See [#78](https://github.com/layer5io/getnighthawk/issues/78). So you might want to add the new questions and answers too.

Reply:
@toth2000 can you pls look into this too?